pkg/cmd/admin/prune/images.go

```diff
@@ -72,7 +77,13 @@ var (
 	  %[1]s %[2]s --prune-over-size-limit
 
 	  # To actually perform the prune operation, the confirm flag must be appended
-	  %[1]s %[2]s --prune-over-size-limit --confirm`)
+	  %[1]s %[2]s --prune-over-size-limit --confirm
+
+	  # Force the insecure http protocol with the particular registry host name
+	  %[1]s %[2]s --registry-url=http://registry.example.org --confirm
```

Comment:
shouldn't this example include the --force-insecure arg?

Reply:
It could, but as the command's help says:

> Insecure connection is allowed in the following cases unless certificate-authority is specified:
> 
>  1. --force-insecure is given  
>  2. provided registry-url is prefixed with http://

Reply:
it wasn't clear to me that those were "either/or" conditions, I assumed they were both required.  (to ensure that someone didn't accidentally use an insecure registry without explicitly intending to do so).

So if it's true that --force-insecure is not a requirement when explicitly indicating http, then i guess I retract my comment.

Reply:
(but maybe add an "or" to the help text)

Reply:
I see, I'll rephrase.